### PR TITLE
[#1610]  doc(lakehouse-iceberg): add Iceberg metrics store document

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
add Iceberg metrics store document

### Why are the changes needed?

Fix: #1610 

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
just document

### CODE REVIEW

File: docs/iceberg-rest-service.md [hunk 1]
```diff
@@ -48,6 +48,17 @@ Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the
...
 Filter parameters can also be specified in the configuration, by setting config entries of the form `gravitino.auxService.iceberg-rest.<class name of filter>.param.<param name>=<value>`
 
+### Iceberg metrics store configuration
+
+Gravitino provides a pluggable metrics store to save and delete Iceberg metrics.
+
+| Configuration item                                         | Description                                                                                                                                                                       | Default value | Required | Since Version |
+|------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `gravitino.auxService.iceberg-rest.metricsStore`           | The storage class name, designed to support the storage of Iceberg metrics, is pluggable and should extend from `com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics`. | (none)        | No       | 0.4.0         |
```
Comment (jerryshao) [inline docs/iceberg-rest-service.md:57]: it is pluggable...

File: docs/iceberg-rest-service.md [hunk 2]
```diff
@@ -48,6 +48,17 @@ Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the
...
+| `gravitino.auxService.iceberg-rest.metricsStoreRetainDays` | The retain days of Iceberg metrics, the value not greater than 0 means retain forever.                                                                                            | -1            | No       | 0.4.0         |
```
Comment (jerryshao) [inline docs/iceberg-rest-service.md:58]: The days to retain Iceberg metrics in store, the value smaller than 0 means retaining forever.

File: docs/iceberg-rest-service.md [hunk 3]
```diff
@@ -48,6 +48,17 @@ Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the
...
+| `gravitino.auxService.iceberg-rest.metricsQueueCapacity`   | The metrics queue capacity is designated for buffering Iceberg metrics before writing to IcebergMetricsStore. If the metrics exceed the queue's capacity, they will be dropped.   | 1000          | No       | 0.4.0         |
```
Comment (jerryshao) [inline docs/iceberg-rest-service.md:59]: The size of queue to store metrics temporally before storing to the persistent storage. Metrics will be dropped when queue is full.

File: docs/iceberg-rest-service.md [hunk 4]
```diff
@@ -48,6 +48,17 @@ Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the
 The filter in the customFilters should be a standard javax servlet Filter.
 Filter parameters can also be specified in the configuration, by setting config entries of the form `gravitino.auxService.iceberg-rest.<class name of filter>.param.<param name>=<value>`
 
+### Iceberg metrics store configuration
+
+Gravitino provides a pluggable metrics store to save and delete Iceberg metrics.
```
[the remark this answers is not in the document]
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: updated

File: docs/iceberg-rest-service.md (hunk 1 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: updated

File: docs/iceberg-rest-service.md (hunk 3 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: updated

**Review by jerryshao — APPROVED** [review]

**Pull request merged**